src/benchmarks/micro/Categories.cs

```diff
@@ -15,12 +15,12 @@ public static class Categories
         /// <summary>
         /// benchmarks belonging to this category are executed for CoreFX CI jobs
         /// </summary>
-        public const string CoreFX = "CoreFX";
+        public const string Libraries = "CoreFX";
```

Comment:
@billwert @DrewScoggins I have changed the folder structure and renamed the categories (CoreFX -> Libraries, CoreCLR -> Runtime) but did not change the... values of the constants.

So in the code we now have:

```cs
[BenchmarkCategory(Categories.Runtime, Categories.Libraries)]
```

But the values remained `CoreFX` and `CoreCLR`. I did that to make sure that we don't change the real categories names which I assume would affect the naming in the Reporting Service. Please let me know if this is good or bad idea.

Reply:
@DrewScoggins Can you confirm whether or not category is part of test identity? I think it is, but this leads to interesting questions (such as if someone adds a category will they break us)?

I think this is a fine change for now, leaving the values the same. I'm a little worried that we're going to have the impetus to change these again the next time we rename teams, and would like to avoid that.

Reply:
We store the test category in a side table, so it is not part of the main identity. Updating the category for the test will just add this category as one of many that are applied to the test. So this would not break anything in the reporting side.

Reply:
Thanks for the explanation. I've changed the values as well. So now CoreCLR and CoreFX names are not really used anymore